openbas-front/src/admin/components/simulations/simulation/ExerciseMainInformation.tsx

```diff
@@ -18,11 +23,18 @@ interface Props {
 const ExerciseMainInformation: React.FC<Props> = ({ exercise }) => {
   const { t } = useFormatter();
   const sortByOrder = R.sortWith([R.ascend(R.prop('phase_order'))]);
+  const scenarioBaseUri = '/admin/scenarios';
+  const scenarioId: string = exercise.exercise_scenario || '';
+  const { scenario } = scenarioId
```

Comment:
If I remove the parent scenario, I get this error on exercise side:
![image](https://github.com/user-attachments/assets/89037bd2-763e-41d3-a927-6e078b1e7f51)
![image](https://github.com/user-attachments/assets/2f404a92-0e24-470f-bde2-c98960b2299c)

Reply:
💀

Reply:
Ok I had made a mistake in null checking, I fixed it in the latest commit